Dshop/apps/products_catalogue/views.py

```diff
@@ -1,3 +1,90 @@
-#from django.shortcuts import render
+from django.views import View
+from lxml import etree
+import xml.etree.ElementTree as ET
+import requests
+from django.http import HttpResponse
+
+from .models import Product, CeneoCategory
+
+
+class ProductListView(View):
```

Comment:
@memorisanka I looked at the conflict. Issue here is it's the same name we already have for products HTML view for browsers. Please rename it to something like CeneoProductsListView or similar. And, of course update all the refences.